### ISSUE TITLE
Should randomize only provide colors with AA / AAA standard contrast?

### ISSUE THREAD

**Issue by nick-ollier** [juxtopposed/realtimecolors#65]
**Is your feature request related to a problem? Please describe.**
I think the current app does a good job of letting you know when color contrast isn't being hit, so I wouldn't necessarily say this is a *problem*.

**Describe the solution you'd like**
I think color randomization could be improved by removing an element of *randomness* and only returning colors that meet AA / AAA color contrast standards.

I think subtly enforcing good accessibility is a good thing and would ultimately be a net positive!  The user has the option to provide / tweak their own colors, meaning the failure state would still be important, if not more so.

As an aside - This may be a bad / intrusive idea, but if the user tries to set colors with failing contrast, would it be a nice idea to prompt them to let them know?

**Describe alternatives you've considered**
I think the alternative is the current behavior.

**Additional context**
No further context is needed - Absolutely love your app, you're doing gods work, keep it up!

**Comment by PositiveUser** [comment]
I think it already does. [Article from docs](https://www.realtimecolors.com/docs/contrast-checker)
